.github/CODEOWNERS

```diff
@@ -5,23 +5,44 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-* @williamfalcon @borda @teddykoker @awaelchli @nateraw @justusschock @tchaton @SeanNaren @ananyahjha93
+* @williamfalcon @borda @tchaton @SeanNaren @awaelchli @justusschock
 
 # Metrics
-/pytorch_lightning/metrics/*  @teddykoker @ananyahjha93 @justusschock
-/tests/metrics/*              @teddykoker @ananyahjha93 @justusschock
+/pytorch_lightning/metrics/   @teddykoker @ananyahjha93 @justusschock
+/tests/metrics/               @teddykoker @ananyahjha93 @justusschock
 /docs/source/metrics.rst      @teddykoker @ananyahjha93 @justusschock
 
 # API
-/pytorch_lightning/callbacks/base.py  @williamfalcon
-/pytorch_lightning/core/datamodule.py @williamfalcon
-/pytorch_lightning/trainer/trainer.py @williamfalcon
-/pytorch_lightning/core/hooks.py     @williamfalcon
-/pytorch_lightning/core/lightning.py  @williamfalcon
+/pytorch_lightning/callbacks/base.py           @williamfalcon
+/pytorch_lightning/core/datamodule.py          @williamfalcon
+/pytorch_lightning/trainer/trainer.py          @williamfalcon @tchaton
+/pytorch_lightning/core/hooks.py               @williamfalcon
+/pytorch_lightning/core/lightning.py           @williamfalcon @tchaton
```

Comment:
Can you add me to /pytorch_lightning/core/optimizer.py

Reply:
pls, do it as a suggestion :]